cfgov/unprocessed/js/modules/multi-select.js

```diff
@@ -0,0 +1,494 @@
+'use strict';
+
+function init() {
+  var _ = function ( elem, opts ) {
+    var self = this;
+
+    this.elem = $( elem );
```

Comment:
Use native dom lookup, it's [well supported](http://caniuse.com/#feat=queryselector).